src/App.js

```diff
@@ -4,34 +4,72 @@ import podcasts from './test_data_space.json'
 import { Route, Switch, useHistory } from 'react-router-dom'
 import DetailsPage from './pages/DetailsPage/DetailsPage'
 import { useState } from 'react'
+import FavoritesPage from './pages/FavoritesPage/FavoritesPage'
 
 function App() {
   const [detailedPodcast, setDetailedPodcast] = useState()
```

Comment:
Is this an array, an object? Would be easier to understand if you'd provide an initial state.